pkg/loop/internal/chain_reader_test.go

```diff
@@ -2,17 +2,19 @@ package internal
 
 import (
 	"errors"
-	"github.com/stretchr/testify/assert"
-	"google.golang.org/grpc/test/bufconn"
 	"net"
 	"sync"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc/test/bufconn"
+
 	"github.com/stretchr/testify/require"
 
 	"github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/pb"
 
 	"context"
```

Comment:
Pre-existing, but there's a std lib context down here?

cc @chainchad seems like the linter is failing in this repo too